### DISCUSSION TITLE
AutoUpdate confirmation ?

### DISCUSSION THREAD

**Discussion by lipici** [bromite/bromite#2046] [General]
What happens if i deselect autoupdate ? Please be more specific.

1 )Filters will still autoupdate ? 
2) Filters and browser will not autoupdate ?
3) Filters will autoupdate but browser will not autoupdate

![Screenshot_2022-05-04_08-29-47](https://user-images.githubusercontent.com/46893370/166627586-8b9e5e49-1259-4cf3-8605-f4f5a43471cd.png)

Browser was checking automatically until now and you see when update its acailabke when you pressed those 3 vertical dots. 
Now if i deselect autoupdate ?

**Comment by uazo** [comment]
indications on the screen seem very clear to me.

**Reply by lipici** [reply]
Sorry, didnt seen "they cannot be disabled".

Can be closed.

**Comment by csagan5** [comment]
The checkbox is only for the Bromite app automatic updates.

**Reply by lipici** [reply]
Yes, its clear from message but i didnt seen a part of text that describes.
